src/pages/SettingsView/SettingsView.tsx

```diff
@@ -49,7 +49,23 @@ export function SettingsView() {
             </FieldSet>
             <FieldSet label="Export" data-testid="export-settings">
               <InlineFieldRow>
-                <InlineField label="Enable flamegraph.com" labelWidth={24}>
+                <InlineField
+                  label="Enable flamegraph.com"
+                  labelWidth={24}
+                  tooltip={
+                    <div className={styles.tooltip}>
+                      <p>
+                        When enabled, this option allows users to export profiles to{' '}
+                        <a href="https://flamegraph.com/" target="_blank" rel="noreferrer noopener">
+                          flamegraph.com
+                        </a>{' '}
+                        and share interactive flamegraphs via <em>a public URL</em>.
+                      </p>
+                      <p>Disable this option if data privacy is a concern.</p>
```

Comment:
Excellent phrasing! I'm going to use your wording in the other PR for the docs. https://github.com/grafana/website/pull/22779/files#r1838468163